src/pwa-buildpack/project-setup/create-simple-peregrine-app/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create simple peregrine app
+---
+
+In the [previous topic], you created configuration files for your theme project.
+In this topic, you will create a simple app that follows the [Peregrine] pattern.
+
+## Create app.js
+
+1. Inside `src`, create a directory called `components`.
+1. Inside the `components` directory, create an `app.js` file with the following content:
+
+   ``` jsx 
+   import React from 'react';
+
+    export default class App extends React.Component {
+        render() {
+            return (
+                <h1>
+                    Hello, Studio!
+                </h1>
+            );
+        }
+    }
+
+   ```
+
+   This code defines a React component that returns "Hello, Studio!" inside `h1` tags.
+
+## Create index.js
+
+Inside `src` create an `index.js` file with the following content:
+
+``` javascript
+import Peregrine from '@magento/peregrine';
+import App from './components/app';
```

Comment:
@jimbo and @DrewML might notice here that I didn't use dynamic imports for this absolutely baseline setup. The Babel and Webpack configs for dynamic import were giving me a bit of trouble, but I was working very fast, so it may have been an obvious thing. Are you all right with proceeding without dynamic import in these basic shell files?